graphql_compiler/schema_generation/sqlalchemy/utils.py

```diff
@@ -1,18 +1,32 @@
 # Copyright 2019-present Kensho Technologies, LLC.
+from typing import Iterable, List, Set
+
+from sqlalchemy import Table
+
 from ..exceptions import MissingPrimaryKeyError
 
 
-def validate_that_tables_have_primary_keys(tables):
+def validate_that_tables_have_primary_keys(tables: Iterable[Table]):
```

Comment:
Missing return type annotation: 
```suggestion
def validate_that_tables_have_primary_keys(tables: Iterable[Table]) -> None:
```